orderbook/src/api/get_fee_info.rs

```diff
@@ -50,7 +50,7 @@ pub fn get_fee_info_response(
             StatusCode::NOT_FOUND,
         )),
         Err(MinFeeCalculationError::UnsupportedToken(token)) => Ok(reply::with_status(
-            super::error("UnsupportedToken", format!("Token address {}", token)),
+            super::error("UnsupportedToken", format!("Token address {:?}", token)),
```

Comment:
Whoops! My mistake. Does this mean the token address wasn't being displayed before?

Reply:
Actually this was my mistake in #550